src/pages/PageTemplate/Sidebar/SidebarStyles.tsx

```diff
@@ -0,0 +1,57 @@
+import Container from '@mui/material/Container';
+//import Tab from '@mui/material/Tab';
+import React from 'react';
+
+
+export const ContainerSidebarWrapper = ({
+  children,
+}: {
+  children?: React.ReactNode;
+}) => (
+  <Container
+    maxWidth='xl'
+    sx={{
+      justifyContent: 'space-between',
+      display: 'flex',
+      flexDirection: 'row',
+      marginBlock: '0.5rem',
+      marginTop: '4rem',
+      marginLeft: '9.4rem',
```

Comment:
you can remove this marginLeft,  does not change the styling much and looks better on smaller screens

Reply:
done